### ISSUE TITLE
1. General design guidelines

### ISSUE THREAD

**Issue by Cobra-Bitcoin** [bitcoin-dot-org/Bitcoin-Wallet#3]
These are the general design guidelines that apply for the entire application, I generally prefer if these are stuck too, but modifications that do something better than what is outlined here are welcomed.

**Window size**
The size of the application is the default window size for Electron applications: a width of 800px and a height of 600px.

**Colors:**
The main colors should be based on the current bitcoin.org website design's darker palette with orange as the contrast and possibly some white where appropriate, but generally it should be quite dark:

- Orange: #FF9500
- Dark: #090c14

**Titlebar**
Use a custom titlebar for all the pages, but instead of being green like the example below, use #090c14. Don't show a sub-menu in the title bar (so no edit, view, window, etc options), just only two options: minimize and close, nothing else.

![](https://raw.githubusercontent.com/Treverix/custom-electron-titlebar/master/screenshots/window_2.png)

**Font**
Same as in the website, use Titillium Web and possibly other new fonts if appropriate. https://fonts.google.com/specimen/Titillium+Web.

**Logo**
Use the standard Bitcoin logo as the main logo of the wallet: 
![](https://upload.wikimedia.org/wikipedia/commons/4/46/Bitcoin.svg)

**Modals**
Ideally all modals will be white, for maximum contrast, as is the case on the website with the donation modal, and when a modal appears, the background will be dimmed. Modals should have an "x" icon at the top right to indicate the modal can be closed.

**Tables**
All tables need to have pagination designed into the table itself in a nice way, to allow for example the user to easily scroll through their transactions.
![](https://i.pinimg.com/originals/c9/77/da/c977da32a5969defcb085fdc69eae11b.png)

**Animations**
If the animation makes sense and helps the user, they are welcomed, but animations that are easy to implement in code are preferred. In particular an animation of when a modal is opened would be useful, along with animations to "slide" the user between different phases of the onboarding process.

**Comment by natiwa** [comment]
Icon. Let me know your thoughts. Do you need it in different sizes as well? 
![App Icon 512x512](https://user-images.githubusercontent.com/29778945/93760705-d74a4f80-fc0c-11ea-8ed4-24a98535f48f.png)

**Comment by Cobra-Bitcoin** [comment]
@natiwa Awesome, it's just what I had in mind. And no, 512x522x is fine, I don't think we'll need it in other sizes.

**Comment by alexcherman** [comment]
Hey @Cobra-Bitcoin, you mentioned that height of the application window should be 600px. Is that including the titlebar at the top?

**Comment by Cobra-Bitcoin** [comment]
@alexcherman Yeah it includes the titlebar.

**Comment by alexcherman** [comment]
Ok, thanks for the info

**Comment by Cobra-Bitcoin** [comment]
@natiwa @alexcherman Also, let's not forget we still need the custom SVG animated loading indicator designed to replace this horrible looking one: https://github.com/bitcoin-dot-org/Bitcoin-Wallet/blob/master/src/assets/ball-triangle.svg.

**Comment by alexcherman** [comment]
@Cobra-Bitcoin Yes, sure. It's already done. @natiwa will send it to you as soon as she has time.

**Comment by natiwa** [comment]
hi,

 @Cobra-Bitcoin Loader will be animated like this one:
![Tripple-Ring-Preloader-1](https://user-images.githubusercontent.com/29778945/95484586-f24ceb80-0990-11eb-9411-ae40ab495c85.gif)
But the design will be like this one:
![Loader](https://user-images.githubusercontent.com/29778945/95484646-02fd6180-0991-11eb-9507-5d64acb65aa1.png)

@alexcherman needs to code it.
Let me know what you think.

**Comment by Cobra-Bitcoin** [comment]
@natiwa Looks really good, I'm happy with it, especially if the animation is smoother.

**Comment by alexcherman** [comment]
Hi @Cobra-Bitcoin Please check updated loading indicator: https://drive.google.com/file/d/1Ll37So364rFBt6Xsh_MRbNZLwK9AWojZ/view?usp=sharing
Let us know what do you think please.

**Comment by Cobra-Bitcoin** [comment]
@alexcherman Looks awesome, really nice job! 👍

**Comment by alexcherman** [comment]
Hey @Cobra-Bitcoin. I have question regarding window size. Can users resize the window?

**Comment by Cobra-Bitcoin** [comment]
@alexcherman Nope, it won't be possible to resize the window.
